include/util/hex_float.h

```diff
@@ -111,9 +135,9 @@ std::istream& operator>>(std::istream& is, FloatProxy<T>& value) {
 template <typename T>
 struct HexFloatTraits {
   // Integer type that can store this hex-float.
-  typedef void uint_type;
+  using uint_type = void;
   // Signed integer type that can store this hex-float.
-  typedef void int_type;
+  using int_type = void;
   // The number of bits that are actually relevant in the uint_type.
```

Comment:
Optional:  Out of symmetry with the previous two, perhaps should also have, along with comments saying what they are.
  using underlying_type = void;
  using native_type = void;